### DISCUSSION TITLE
Device still hidden, even though i disabled it and reinstalled HidHide

### DISCUSSION THREAD

**Discussion by Funky-Punky** [nefarius/HidHide#135] [Q&A] [answered]
Okay, this is going to be a ramble. Relax, lay back and grab a drink.

I use HidHide, vJoy and Joystick gremlin to micromanage my experience in multiple games with my steering wheel. Yesterday i noticed, that my vanilla xbox one controller wasn't working any more.

I tried my controller on other machines and a borrowed controller on my machine. Everything worked, except for my controller on my machine. I also reinstalled HidHide multiple times, upgraded the controllers firmware and disabled vJoy to narrow down the problem. And that brings me to my current situation:
The borrowed controller works beautifully with every software. I block it in HidHide - I cant see it in steam or on https://hardwaretester.com/gamepad. I don't block it - I can use it everywhere.
But my own controller doesn't behave like that. It behaves as it it was still blocked by Hidhide. (even if HidHide is not even currently installed). Steam doesn't see it, just as any other app. But if i use joystick gremlin to remap the inputs to a vJoy gamepad, everything works. When i use [joy.cpl](https://kanuan.github.io/DS4WSite/troubleshooting/controllers-check/#windows-game-controllers-list), I can see every movement and "Microsoft apps" like "Settings" or "[Xbox Accessories](https://www.microsoft.com/p/xbox-accessories/9nblggh30xj3?)" in general work, and can use the gamepad.

So my question is: Is it possible, that i somehow broke HidHide so hard, that uninstalling it doesn't even stop its effect? How would i fix this? Can i purge everything HidHide related of my pc and reinstall it? Is my controller just plain broken?

Thanks for reading and if you need any other Information, don't hesitate to ask.

**Comment by nefarius** [comment, marked as answer]
Fire up Windows Terminal/PowerShell and run:

```PowerShell
Get-ItemPropertyValue -Path 'HKLM:\SYSTEM\CurrentControlSet\Control\Class\{05f5cfe2-4733-4950-a6bb-07aad01a3a84}' -Name UpperFilters
Get-ItemPropertyValue -Path 'HKLM:\SYSTEM\CurrentControlSet\Control\Class\{745a17a0-74d3-11d0-b6fe-00a0c90f57da}' -Name UpperFilters
Get-ItemPropertyValue -Path 'HKLM:\SYSTEM\CurrentControlSet\Control\Class\{d61ca365-5af4-4486-998b-9db4734c6ca3}' -Name UpperFilters
```

if all of those come back empty, HidHide is definitely not in the picture anymore. Output of a normal system with HH installed:

![image](https://github.com/nefarius/HidHide/assets/286631/f7664cf1-2338-462c-9896-43dd8e4d994d)

**Reply by Funky-Punky** [reply]
All three lines are empty, I'm afraid. Seems like my problem is somewhere else.
Thanks for taking the time.

**Reply by nefarius** [reply]
Well at least you could rule out one thing, good luck!

**Reply by Funky-Punky** [reply]
![image](https://github.com/nefarius/HidHide/assets/42394930/3d6114ec-3ffb-4d55-a244-25eeacd8c015)

**Comment by nefarius** [comment]
Also running [Legacinator](https://github.com/nefarius/Legacinator) can't hurt in this case.

**Reply by Funky-Punky** [reply]
It removed something, but it still doesn't work :,(
